docs/src/pages/landing-page/Index.vue

```diff
@@ -134,13 +134,55 @@
           <div class="lp-heading lp-heading-large">
             What our community thinks of quasar
           </div>
-          <div class="text-size-20 text-lp-light text-weight-medium">
+          <div class="lp-heading lp-heading-small">
             "You've never heard of Quasar? It's the framework that made a mobile
-            app, desktop app and browser extension altogether in less than
-            12minutes."
+            app, desktop app and browser extension altogether in less than 12 minutes."
           </div>
         </div>
       </div>
+      <q-carousel
+        v-model="slide"
+        transition-prev="scale"
+        transition-next="scale"
+        swipeable
+        animated
+        navigation
+        :autoplay="true"
+        padding
+        arrows
+        height="400px"
+        class="bg-transparent rounded-borders"
+      >
+        <template v-slot:navigation-icon="{ active, btnProps, onClick }">
+          <q-btn
+            v-if="active"
+            size="xs"
+            :icon="btnProps.icon"
+            color="lp-primary"
+            flat
+            round
+            dense
+            @click="onClick"
+          />
+          <q-btn
+            v-else
+            size="6px"
+            :icon="btnProps.icon"
+            color="grey-8"
+            flat
+            round
+            dense
+            @click="onClick"
+          />
+        </template>
+        <q-carousel-slide :name="i" class="showcase-cards" v-for="(showcaseContent, i) in showcaseContents" :key="i">
+          <div class="my-grid">
+            <div style="width: 90%" v-for="(showcases, i) in showcaseContent" :key="i">
```

Comment:
Let aside the classic "don't use inline styles", why is this needed? You can control this via CSS grid properties if needed, but in this case it seems fine even if you just remove this inline style

Reply:
Try to use expanded and meaninful variable names where possible, in this case `index` instead of `i`
Also, you're shadowing the previous `i` iterator variable, and creating multiple element with the same key
Possibly differentiate them using template strings: ``:key="`slide-${slideIndex}`"`` and ``:key="`twitter-card-${cardIndex}`"``

Reply:
Why is this div needed, btw?
Same for the container div, you could just add `my-grid` into the `q-carousel-slide` tag